content/manuals/compose/bridge/_index.md

```diff
@@ -8,22 +8,31 @@ weight: 50
 
 {{< summary-bar feature_name="Compose bridge" >}}
 
-Compose Bridge converts your Docker Compose configuration into platform-specific formats—primarily Kubernetes manifests. The default transformation generates Kubernetes manifests and a Kustomize overlay which are designed for deployment on Docker Desktop with Kubernetes enabled.  
+Compose Bridge converts your Docker Compose configuration into platform-specific deployment formats such as Kubernetes manifests. By default, it geneterates:
 
-It's a flexible tool that lets you either take advantage of the [default transformation](usage.md) or [create a custom transformation](customize.md) to suit specific project needs and requirements.  
+- Kubernetes manifests 
+- A Kustomize overlay
 
-Compose Bridge significantly simplifies the transition from Docker Compose to Kubernetes, making it easier for you to leverage the power of Kubernetes while maintaining the simplicity and efficiency of Docker Compose.
+These outputs are ready for deployment on Docker Desktop with [Kubernetes enabled](/manuals/desktop/settings-and-maintenance/settings.md#kubernetes).  
+
+Compose Bridge helps you bridge the gap between Compose and Kubernetes, making it easier to adopt Kubernetes while keeping the simplicity and efficiency of Compose.
+
+It's a flexible tool that lets you either take advantage of the [default transformation](usage.md) or [create a custom transformation](customize.md) to suit specific project needs and requirements. 
 
 ## How it works
 
-Compose Bridge uses transformations to let you convert a Compose model into another form. 
+Compose Bridge uses transformations to convert a Compose model into another form. 
 
 A transformation is packaged as a Docker image that receives the fully resolved Compose model as `/in/compose.yaml` and can produce any target format file under `/out`.
 
 Compose Bridge provides its own transformation for Kubernetes using Go templates, so that it is easy to extend for customization by replacing or appending your own templates.
 
 For more detailed information on how these transformations work and how you can customize them for your projects, see [Customize](customize.md).
 
+With Composer version X, Compose Bridge now also supports applications that use LLMs via Docker Model Runner.
```

Comment:
Todo: fill in X

Reply:
It's not link to Compose version, those are images used by Bridge, which means it's already available to everyone (it was added to https://github.com/docker/compose-bridge-transformer/releases/tag/v0.0.3)